src/main/java/com/ogp/configurator/ConfigService.java

```diff
@@ -236,18 +260,18 @@ public Object getConfigEntity(Class<Object> clazz, String key) {
 		final List<Object> configEntities = new ArrayList<>();
 		
 		if (configObjects.containsKey(classFromType)) {
-			System.out.printf("Config cache for class %s have %d elements\n", classFromType.toString(), configObjects.get(classFromType).size());
+			logger.debug("Config cache for class %s have %d elements\n", classFromType.toString(), configObjects.get(classFromType).size());
 			Map<String, Object> entities = configObjects.get(classFromType);
 			synchronized (entities) {
 				configEntities.addAll(entities.values());
 			}
 		} else {
-			System.out.printf("Class %s not found in cache\n", classFromType);
+			logger.debug("Class %s not found in cache\n", classFromType);
```

Comment:
Does it work like this? [maybe should be - logger.debug("Class {} not found in cache", classFromType); ]